app/src/main/res/values/strings.xml

```diff
@@ -20,9 +20,9 @@
 <resources>
     <string name="___app_was_not_found">%s was not found!</string>
     <string name="about">About</string>
-    <string name="about_subtext">Hi\nMy name is Uriah Shaul Mandel, and i\'m the developer of this platform.\nI live in
-        Tel Aviv.\nIv\'e spent a lot of time and effort to make this platform, and released it for free, for everyone to
-        use\nyou are more than welcome to contact me, by email.\nbaldphone.contact@gmail.com
+    <string name="about_subtext">Hi!\nMy name is Uriah Shaul Mandel, and I\'m the developer of this platform.\nI live in
+        Tel Aviv.\nI\'ve spent a lot of time and effort to make this platform, and released it for free, for everyone to
+        use.\nYou are more than welcome to contact me, by email.\nbaldphone.contact@gmail.com
```

Comment:
This sentence sounds a little rough, but I don't want to make major changes since it's written in the first person... though maybe that's not the best principle, since this string will have to be translated into other languages...

> Hi! My name is Uriah Shaul Mandel, and I'm the lead developer of this platform.
> I'm from ____, and I currently live in Tel Aviv.*
> I've spend a lot of time and effort making this platform, and I've released it for free, for all to use.
> You're more than welcome to contact me by email:
> baldphone.contact@gmail.com

`*` The original second sentence sounds a little choppy. Alternatives might be
> I live in Tel Aviv, Israel, which is where I grew up.
or
> I'm from Israel, and have lived in Tel Aviv for most of my life*

`*` maybe say `most of my adult life`

But I'm not your biographer, so I don't feel qualified to make any of these changes.